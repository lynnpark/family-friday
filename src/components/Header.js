import React from "react";
import Logo from "./Logo"

export default function Header() {
  return (
    <div className="header">
      <div className="topNav">
        <Logo />
        <span>by Lynn Park</span>
      </div>
    </div>
  );
}
