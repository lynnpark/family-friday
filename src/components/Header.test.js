import React from "react";
import { render } from "@testing-library/react";
import Header from "./Header";

describe("Header Component", () => {
  test("Matches Snapshot", () => {
    const component = render(<Header />);
    expect(component).toMatchSnapshot();
  });
});
