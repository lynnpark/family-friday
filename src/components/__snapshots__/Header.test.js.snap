// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component Matches Snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="header"
      >
        <div
          class="topNav"
        >
          <svg
            alt="Apartment List"
            class="logo"
            viewBox="0 0 1085.7 178.1"
            xmlns="http://www.w3.org/2000/svg"
          >
            <g
              data-name="Layer 2"
            >
              <g
                data-name="Layer 1"
              >
                <path
                  d="M86.3 127.4H73.8a1.4 1.4 0 0 1-1.3-.9l-8.2-21.1a1.4 1.4 0 0 0-1.3-.9H24.2a1.4 1.4 0 0 0-1.3.9l-8.2 21.1a1.4 1.4 0 0 1-1.3.9h-12a1.4 1.4 0 0 1-1.3-1.9l34.4-87a1.4 1.4 0 0 1 1.3-.9H52a1.4 1.4 0 0 1 1.3.9l34.4 87a1.4 1.4 0 0 1-1.4 1.9zM28.6 90.8h30L43.6 52zM157.8 96.7c0 18.8-13.5 32.3-30.7 32.3-8.7 0-15-3.5-19.2-8.9v31.5a1.4 1.4 0 0 1-1.4 1.4H95.9a1.4 1.4 0 0 1-1.4-1.4V67.4a1.4 1.4 0 0 1 1.4-1.4h10.5a1.4 1.4 0 0 1 1.4 1.4v5.9c4.2-5.4 10.5-8.9 19.2-8.9 17.3 0 30.8 13.5 30.8 32.3zm-50.9 0c0 11.2 7.4 19.7 18.5 19.7s18.6-8.9 18.6-19.7S137 77 125.5 77s-18.6 8.5-18.6 19.7zM225.9 127.4h-10.6a1.4 1.4 0 0 1-1.4-1.4v-6c-4.2 5.4-10.6 9-19.3 9-17.2 0-30.7-13.5-30.7-32.3s13.5-32.3 30.7-32.3c8.7 0 15.1 3.6 19.3 9v-6a1.4 1.4 0 0 1 1.4-1.4h10.6a1.4 1.4 0 0 1 1.4 1.4V126a1.4 1.4 0 0 1-1.4 1.4zm-48.2-30.7c0 10.8 7 19.7 18.5 19.7s18.5-8.5 18.5-19.7S207.3 77 196.2 77s-18.5 8.9-18.5 19.7zM277.2 67.3l-2.5 10.5a1.4 1.4 0 0 1-1.9.9 25.5 25.5 0 0 0-8.7-1.6c-8.5 0-14.6 6-14.6 16.5V126a1.4 1.4 0 0 1-1.4 1.4h-10.7a1.4 1.4 0 0 1-1.4-1.4V67.4a1.4 1.4 0 0 1 1.4-1.4h10.5a1.4 1.4 0 0 1 1.4 1.4v5.5c4-5.8 10.3-8.5 17.8-8.5a27.9 27.9 0 0 1 9.1 1.3 1.4 1.4 0 0 1 1 1.6zM332.7 78.2h-22.2v26.7c0 8.3 4.4 11.5 9.7 11.5a23.6 23.6 0 0 0 11.2-3.5 1.3 1.3 0 0 1 1.8.6l4.2 8.4a1.3 1.3 0 0 1-.5 1.7A32.8 32.8 0 0 1 319 129c-13.9 0-22.1-7.5-22.1-23.8v-27h-10.8a1.3 1.3 0 0 1-1.3-1.3v-9.6a1.3 1.3 0 0 1 1.3-1.3H297V48.7a1.3 1.3 0 0 1 1.3-1.3h10.8a1.3 1.3 0 0 1 1.3 1.3V66h22.2a1.3 1.3 0 0 1 1.3 1.3v9.6a1.3 1.3 0 0 1-1.2 1.3zM445.7 88.8V126a1.4 1.4 0 0 1-1.4 1.4h-10.7a1.4 1.4 0 0 1-1.4-1.4V93.2c0-10.3-5.3-16.1-14.3-16.1S402 82.8 402 93.4V126a1.4 1.4 0 0 1-1.4 1.4H390a1.4 1.4 0 0 1-1.4-1.4V93.2c0-10.3-5.3-16.1-14.3-16.1s-15.9 5.7-15.9 16.2V126a1.4 1.4 0 0 1-1.4 1.4h-10.7a1.4 1.4 0 0 1-1.4-1.4V67.4a1.4 1.4 0 0 1 1.4-1.4h10.5a1.4 1.4 0 0 1 1.4 1.4v6.1c4.9-7.2 12.6-9.1 18.8-9.1 9.4 0 17 4.2 21.2 11 5.3-8.8 14.8-11 22.4-11 14.8 0 25.1 10.1 25.1 24.4zM511.8 96.6c0 1.3-.1 2.5-.2 3.7a1.4 1.4 0 0 1-1.4 1.3h-44c1.8 11 9.6 15.6 18.3 15.6a29.2 29.2 0 0 0 16.7-5.6 1.3 1.3 0 0 1 1.9.3l5.2 7.4a1.4 1.4 0 0 1-.3 1.8c-7.3 5.8-15.4 7.9-24.3 7.9-18.4 0-31.4-12.8-31.4-32.3s12.5-32.3 30.6-32.3 28.9 13.1 28.9 32.2zm-45.6-5.2h31.9c-1.5-9.6-7.1-15.1-15.6-15.1s-14.9 6.2-16.3 15.1zM575.4 88.8V126a1.4 1.4 0 0 1-1.4 1.4h-10.7a1.4 1.4 0 0 1-1.4-1.4V93.2c0-10.3-5.3-16.1-14.3-16.1s-15.9 5.7-15.9 16.2V126a1.4 1.4 0 0 1-1.4 1.4h-10.6a1.4 1.4 0 0 1-1.4-1.4V67.4a1.4 1.4 0 0 1 1.4-1.4h10.6a1.4 1.4 0 0 1 1.4 1.4v6.1c4.9-7.2 12.6-9.1 18.8-9.1 14.6 0 24.9 10.1 24.9 24.4z"
                  fill="#210d36"
                />
                <path
                  d="M628.8 78.2h-22.1v26.7c0 8.3 4.4 11.5 9.7 11.5a23.5 23.5 0 0 0 11.1-3.4 1.4 1.4 0 0 1 1.9.6l4.1 8.3a1.4 1.4 0 0 1-.5 1.8 32.8 32.8 0 0 1-17.8 5.4c-13.9 0-22.1-7.5-22.1-23.8v-27h-10.8a1.4 1.4 0 0 1-1.4-1.4v-9.5a1.4 1.4 0 0 1 1.4-1.4h10.9V48.8a1.4 1.4 0 0 1 1.4-1.4h10.7a1.4 1.4 0 0 1 1.4 1.4V66h22.1a1.4 1.4 0 0 1 1.4 1.4v9.4a1.4 1.4 0 0 1-1.4 1.4z"
                  fill="#210d36"
                />
                <path
                  d="M942.9 127.4h-61.4a1.4 1.4 0 0 1-1.4-1.4V39.1a1.4 1.4 0 0 1 1.4-1.4h11.7a1.4 1.4 0 0 1 1.4 1.4v74.6h48.3a1.4 1.4 0 0 1 1.4 1.4V126a1.4 1.4 0 0 1-1.4 1.4zM969 44.7c0 6.1-5.5 10.6-11.9 9.1a9.7 9.7 0 0 1-6.6-6.6c-1.5-6.5 3-12 9.1-11.9a9.8 9.8 0 0 1 9.4 9.4zm-4.1 82.7h-10.6a1.4 1.4 0 0 1-1.4-1.4V67.4a1.4 1.4 0 0 1 1.4-1.4h10.6a1.4 1.4 0 0 1 1.4 1.4V126a1.4 1.4 0 0 1-1.4 1.4zM1021.6 80.1a1.4 1.4 0 0 1-1.9.6 40.2 40.2 0 0 0-17.7-4.5c-7.3 0-11.6 2.9-11.6 7.3s4.5 5.5 10.1 6.3l6.3 1c13.4 1.9 21.3 7.6 21.3 18.1s-10.3 20.1-28 20.1c-6.1 0-16.5-1.1-25.3-7.1a1.4 1.4 0 0 1-.4-1.9l4.7-7.6a1.4 1.4 0 0 1 2-.4c4.3 3 9.9 5.4 19 5.4s14-2.9 14-7.5-3.3-5.8-10.6-6.8l-6.4-.8c-13.6-1.9-20.8-8.1-20.8-18 0-12.2 9.7-19.7 25.4-19.7 9 0 17.5 2.1 23.7 5.8a1.4 1.4 0 0 1 .5 1.9zM1080.9 78.2h-22.2v26.7c0 8.3 4.4 11.5 9.7 11.5a23.5 23.5 0 0 0 11.1-3.4 1.4 1.4 0 0 1 1.9.6l4.1 8.3a1.4 1.4 0 0 1-.5 1.8 32.8 32.8 0 0 1-17.8 5.4c-13.9 0-22.1-7.5-22.1-23.8v-27h-10.9a1.4 1.4 0 0 1-1.4-1.4v-9.5a1.4 1.4 0 0 1 1.4-1.4h10.9V48.8a1.4 1.4 0 0 1 1.4-1.4h10.7a1.4 1.4 0 0 1 1.4 1.4V66h22.2a1.4 1.4 0 0 1 1.4 1.4v9.4a1.4 1.4 0 0 1-1.3 1.4zM721.6 112.8V66.4l-12.9 7.5v46.4l12.9-7.5z"
                  fill="#210d36"
                />
                <g
                  fill="#210d36"
                >
                  <path
                    d="M802 162.3a6.5 6.5 0 0 0 3.2-5.6V92.8a6.5 6.5 0 0 0-.9-3.2l-41.2-71.5a6.5 6.5 0 0 0-8.8-2.4 6.5 6.5 0 0 0-2.4 8.8L791.4 93a6.5 6.5 0 0 1 .9 3.2v60.5a6.5 6.5 0 0 0 9.7 5.6z"
                  />
                  <path
                    d="M728.5 30.6l-41.3 23.8A6.5 6.5 0 0 0 684 60v75.2l13-7.5V67.5a6.5 6.5 0 0 1 3.2-5.6l23.5-13.6a6.5 6.5 0 0 1 8.8 2.4l33 57.2a6.5 6.5 0 0 1 .9 3.2v60.5a6.5 6.5 0 0 0 9.7 5.6 6.5 6.5 0 0 0 3.2-5.6v-63.9a6.5 6.5 0 0 0-.9-3.2L737.3 33a6.5 6.5 0 0 0-8.8-2.4zM827.7 147.5a6.5 6.5 0 0 0 3.2-5.6v-64a6.5 6.5 0 0 0-.9-3.2L788.9 3.2A6.5 6.5 0 0 0 780 .9a6.5 6.5 0 0 0-2.4 8.8l39.5 68.4a6.5 6.5 0 0 1 .9 3.2v60.5a6.5 6.5 0 0 0 9.7 5.7z"
                  />
                </g>
                <path
                  d="M696.9 127.7l-13 7.5 59.9 34.6a6.5 6.5 0 0 0 9.7-5.6v-3.7zM708.7 120.3l12.9-7.5 32 18.9v15l-44.9-26.4z"
                  fill="#8434d6"
                />
              </g>
            </g>
          </svg>
          <span>
            by Lynn Park
          </span>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="header"
    >
      <div
        class="topNav"
      >
        <svg
          alt="Apartment List"
          class="logo"
          viewBox="0 0 1085.7 178.1"
          xmlns="http://www.w3.org/2000/svg"
        >
          <g
            data-name="Layer 2"
          >
            <g
              data-name="Layer 1"
            >
              <path
                d="M86.3 127.4H73.8a1.4 1.4 0 0 1-1.3-.9l-8.2-21.1a1.4 1.4 0 0 0-1.3-.9H24.2a1.4 1.4 0 0 0-1.3.9l-8.2 21.1a1.4 1.4 0 0 1-1.3.9h-12a1.4 1.4 0 0 1-1.3-1.9l34.4-87a1.4 1.4 0 0 1 1.3-.9H52a1.4 1.4 0 0 1 1.3.9l34.4 87a1.4 1.4 0 0 1-1.4 1.9zM28.6 90.8h30L43.6 52zM157.8 96.7c0 18.8-13.5 32.3-30.7 32.3-8.7 0-15-3.5-19.2-8.9v31.5a1.4 1.4 0 0 1-1.4 1.4H95.9a1.4 1.4 0 0 1-1.4-1.4V67.4a1.4 1.4 0 0 1 1.4-1.4h10.5a1.4 1.4 0 0 1 1.4 1.4v5.9c4.2-5.4 10.5-8.9 19.2-8.9 17.3 0 30.8 13.5 30.8 32.3zm-50.9 0c0 11.2 7.4 19.7 18.5 19.7s18.6-8.9 18.6-19.7S137 77 125.5 77s-18.6 8.5-18.6 19.7zM225.9 127.4h-10.6a1.4 1.4 0 0 1-1.4-1.4v-6c-4.2 5.4-10.6 9-19.3 9-17.2 0-30.7-13.5-30.7-32.3s13.5-32.3 30.7-32.3c8.7 0 15.1 3.6 19.3 9v-6a1.4 1.4 0 0 1 1.4-1.4h10.6a1.4 1.4 0 0 1 1.4 1.4V126a1.4 1.4 0 0 1-1.4 1.4zm-48.2-30.7c0 10.8 7 19.7 18.5 19.7s18.5-8.5 18.5-19.7S207.3 77 196.2 77s-18.5 8.9-18.5 19.7zM277.2 67.3l-2.5 10.5a1.4 1.4 0 0 1-1.9.9 25.5 25.5 0 0 0-8.7-1.6c-8.5 0-14.6 6-14.6 16.5V126a1.4 1.4 0 0 1-1.4 1.4h-10.7a1.4 1.4 0 0 1-1.4-1.4V67.4a1.4 1.4 0 0 1 1.4-1.4h10.5a1.4 1.4 0 0 1 1.4 1.4v5.5c4-5.8 10.3-8.5 17.8-8.5a27.9 27.9 0 0 1 9.1 1.3 1.4 1.4 0 0 1 1 1.6zM332.7 78.2h-22.2v26.7c0 8.3 4.4 11.5 9.7 11.5a23.6 23.6 0 0 0 11.2-3.5 1.3 1.3 0 0 1 1.8.6l4.2 8.4a1.3 1.3 0 0 1-.5 1.7A32.8 32.8 0 0 1 319 129c-13.9 0-22.1-7.5-22.1-23.8v-27h-10.8a1.3 1.3 0 0 1-1.3-1.3v-9.6a1.3 1.3 0 0 1 1.3-1.3H297V48.7a1.3 1.3 0 0 1 1.3-1.3h10.8a1.3 1.3 0 0 1 1.3 1.3V66h22.2a1.3 1.3 0 0 1 1.3 1.3v9.6a1.3 1.3 0 0 1-1.2 1.3zM445.7 88.8V126a1.4 1.4 0 0 1-1.4 1.4h-10.7a1.4 1.4 0 0 1-1.4-1.4V93.2c0-10.3-5.3-16.1-14.3-16.1S402 82.8 402 93.4V126a1.4 1.4 0 0 1-1.4 1.4H390a1.4 1.4 0 0 1-1.4-1.4V93.2c0-10.3-5.3-16.1-14.3-16.1s-15.9 5.7-15.9 16.2V126a1.4 1.4 0 0 1-1.4 1.4h-10.7a1.4 1.4 0 0 1-1.4-1.4V67.4a1.4 1.4 0 0 1 1.4-1.4h10.5a1.4 1.4 0 0 1 1.4 1.4v6.1c4.9-7.2 12.6-9.1 18.8-9.1 9.4 0 17 4.2 21.2 11 5.3-8.8 14.8-11 22.4-11 14.8 0 25.1 10.1 25.1 24.4zM511.8 96.6c0 1.3-.1 2.5-.2 3.7a1.4 1.4 0 0 1-1.4 1.3h-44c1.8 11 9.6 15.6 18.3 15.6a29.2 29.2 0 0 0 16.7-5.6 1.3 1.3 0 0 1 1.9.3l5.2 7.4a1.4 1.4 0 0 1-.3 1.8c-7.3 5.8-15.4 7.9-24.3 7.9-18.4 0-31.4-12.8-31.4-32.3s12.5-32.3 30.6-32.3 28.9 13.1 28.9 32.2zm-45.6-5.2h31.9c-1.5-9.6-7.1-15.1-15.6-15.1s-14.9 6.2-16.3 15.1zM575.4 88.8V126a1.4 1.4 0 0 1-1.4 1.4h-10.7a1.4 1.4 0 0 1-1.4-1.4V93.2c0-10.3-5.3-16.1-14.3-16.1s-15.9 5.7-15.9 16.2V126a1.4 1.4 0 0 1-1.4 1.4h-10.6a1.4 1.4 0 0 1-1.4-1.4V67.4a1.4 1.4 0 0 1 1.4-1.4h10.6a1.4 1.4 0 0 1 1.4 1.4v6.1c4.9-7.2 12.6-9.1 18.8-9.1 14.6 0 24.9 10.1 24.9 24.4z"
                fill="#210d36"
              />
              <path
                d="M628.8 78.2h-22.1v26.7c0 8.3 4.4 11.5 9.7 11.5a23.5 23.5 0 0 0 11.1-3.4 1.4 1.4 0 0 1 1.9.6l4.1 8.3a1.4 1.4 0 0 1-.5 1.8 32.8 32.8 0 0 1-17.8 5.4c-13.9 0-22.1-7.5-22.1-23.8v-27h-10.8a1.4 1.4 0 0 1-1.4-1.4v-9.5a1.4 1.4 0 0 1 1.4-1.4h10.9V48.8a1.4 1.4 0 0 1 1.4-1.4h10.7a1.4 1.4 0 0 1 1.4 1.4V66h22.1a1.4 1.4 0 0 1 1.4 1.4v9.4a1.4 1.4 0 0 1-1.4 1.4z"
                fill="#210d36"
              />
              <path
                d="M942.9 127.4h-61.4a1.4 1.4 0 0 1-1.4-1.4V39.1a1.4 1.4 0 0 1 1.4-1.4h11.7a1.4 1.4 0 0 1 1.4 1.4v74.6h48.3a1.4 1.4 0 0 1 1.4 1.4V126a1.4 1.4 0 0 1-1.4 1.4zM969 44.7c0 6.1-5.5 10.6-11.9 9.1a9.7 9.7 0 0 1-6.6-6.6c-1.5-6.5 3-12 9.1-11.9a9.8 9.8 0 0 1 9.4 9.4zm-4.1 82.7h-10.6a1.4 1.4 0 0 1-1.4-1.4V67.4a1.4 1.4 0 0 1 1.4-1.4h10.6a1.4 1.4 0 0 1 1.4 1.4V126a1.4 1.4 0 0 1-1.4 1.4zM1021.6 80.1a1.4 1.4 0 0 1-1.9.6 40.2 40.2 0 0 0-17.7-4.5c-7.3 0-11.6 2.9-11.6 7.3s4.5 5.5 10.1 6.3l6.3 1c13.4 1.9 21.3 7.6 21.3 18.1s-10.3 20.1-28 20.1c-6.1 0-16.5-1.1-25.3-7.1a1.4 1.4 0 0 1-.4-1.9l4.7-7.6a1.4 1.4 0 0 1 2-.4c4.3 3 9.9 5.4 19 5.4s14-2.9 14-7.5-3.3-5.8-10.6-6.8l-6.4-.8c-13.6-1.9-20.8-8.1-20.8-18 0-12.2 9.7-19.7 25.4-19.7 9 0 17.5 2.1 23.7 5.8a1.4 1.4 0 0 1 .5 1.9zM1080.9 78.2h-22.2v26.7c0 8.3 4.4 11.5 9.7 11.5a23.5 23.5 0 0 0 11.1-3.4 1.4 1.4 0 0 1 1.9.6l4.1 8.3a1.4 1.4 0 0 1-.5 1.8 32.8 32.8 0 0 1-17.8 5.4c-13.9 0-22.1-7.5-22.1-23.8v-27h-10.9a1.4 1.4 0 0 1-1.4-1.4v-9.5a1.4 1.4 0 0 1 1.4-1.4h10.9V48.8a1.4 1.4 0 0 1 1.4-1.4h10.7a1.4 1.4 0 0 1 1.4 1.4V66h22.2a1.4 1.4 0 0 1 1.4 1.4v9.4a1.4 1.4 0 0 1-1.3 1.4zM721.6 112.8V66.4l-12.9 7.5v46.4l12.9-7.5z"
                fill="#210d36"
              />
              <g
                fill="#210d36"
              >
                <path
                  d="M802 162.3a6.5 6.5 0 0 0 3.2-5.6V92.8a6.5 6.5 0 0 0-.9-3.2l-41.2-71.5a6.5 6.5 0 0 0-8.8-2.4 6.5 6.5 0 0 0-2.4 8.8L791.4 93a6.5 6.5 0 0 1 .9 3.2v60.5a6.5 6.5 0 0 0 9.7 5.6z"
                />
                <path
                  d="M728.5 30.6l-41.3 23.8A6.5 6.5 0 0 0 684 60v75.2l13-7.5V67.5a6.5 6.5 0 0 1 3.2-5.6l23.5-13.6a6.5 6.5 0 0 1 8.8 2.4l33 57.2a6.5 6.5 0 0 1 .9 3.2v60.5a6.5 6.5 0 0 0 9.7 5.6 6.5 6.5 0 0 0 3.2-5.6v-63.9a6.5 6.5 0 0 0-.9-3.2L737.3 33a6.5 6.5 0 0 0-8.8-2.4zM827.7 147.5a6.5 6.5 0 0 0 3.2-5.6v-64a6.5 6.5 0 0 0-.9-3.2L788.9 3.2A6.5 6.5 0 0 0 780 .9a6.5 6.5 0 0 0-2.4 8.8l39.5 68.4a6.5 6.5 0 0 1 .9 3.2v60.5a6.5 6.5 0 0 0 9.7 5.7z"
                />
              </g>
              <path
                d="M696.9 127.7l-13 7.5 59.9 34.6a6.5 6.5 0 0 0 9.7-5.6v-3.7zM708.7 120.3l12.9-7.5 32 18.9v15l-44.9-26.4z"
                fill="#8434d6"
              />
            </g>
          </g>
        </svg>
        <span>
          by Lynn Park
        </span>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
